run/README.namelist

```diff
@@ -686,6 +689,7 @@ Namelist variables for controlling the adaptive time step option:
                                      = 3, RUC land-surface model
                                      = 4, Noah-MP land-surface model (see additional &noah_mp namelist)
                                      = 5, Community Land Model version 4 (CLM4), adapted from CAM
+				     = 6, Community Terrestrial Systems Model (CTSM)
```

Comment:
@kkeene44 Here too. Please also add something like 'See doc/README.CTSM on how to use this option', since this is run using a coupler.

Reply:
Okay, I added that.